pkg/asset/cluster/cluster.go

```diff
@@ -16,13 +16,15 @@ import (
 	"github.com/openshift/installer/pkg/asset/cluster/openstack"
 	"github.com/openshift/installer/pkg/asset/installconfig"
 	"github.com/openshift/installer/pkg/asset/kubeconfig"
+	"github.com/openshift/installer/pkg/asset/password"
 	"github.com/openshift/installer/pkg/terraform"
 	"github.com/openshift/installer/pkg/types"
 )
 
 const (
 	// metadataFileName is name of the file where clustermetadata is stored.
-	metadataFileName = "metadata.json"
+	metadataFileName          = "metadata.json"
+	kubeadminPasswordFileName = "kubeadmin-password"
```

Comment:
I wasn't clear on why `kubeadminPassword` was handled differently from other config until I read this comment: https://github.com/openshift/installer/pull/771#discussion_r238495716
Maybe we can add a comment or something to make it clear?

Reply:
I'll have a follow-up PR to log the password at the end of the install, as well as log the console url.  I'll add that comment to the follow-up PR.